.github/workflows/sphinxbuild.yml

```diff
@@ -17,6 +17,14 @@ jobs:
         docs-folder: "user_manual/"
         pre-build-command: pip install -r requirements.txt
         build-command: make html
+    - name: Pack  the results in local tar file
+      shell: bash
+      run: tar czf /tmp/documentation.tar.gz -C user_manual/_build/html .
+    - name: Upload static documentation
+      uses: actions/upload-artifact@v2.2.4
+      with:
+        name: User manual.zip
```

Comment:
Probably misleading to name it `.zip` when it is actually a gzipped tarball?

Reply:
It is a ZIP file that contains a `.tar.gz` file. The reason is that every single file needs to be uploaded by the action to an API endpoint. While this is no big problem in general from the perspective of programming effort (there is a GitHub action ready to use) it takes a few dozen seconds to upload all generated files. If packed into a `tar.gz` file before, the packing takes ~1-2 sec plus once the API of something in the same range.

Reply:
Ah I see, the zip file is dynamically created when downloading an artefact, containing all uploaded files. A bit strange to have hence a doubled compressed download, but of course it makes sense to not upload all files combined and compressed 👍.